raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -434,6 +430,69 @@ def test_fee_round_trip(flat_fee, prop_fee, imbalance_fee, amount, balance1, bal
     assert amount_with_margin_after_fees >= amount
 
 
+@example(flat_fee=0, prop_fee=0, imbalance_fee=1277, amount=1, balance1=33, balance2=481)
+@given(
+    integers(min_value=0, max_value=100),
+    integers(min_value=0, max_value=10_000),
+    integers(min_value=0, max_value=50_000),
+    integers(min_value=1, max_value=90_000_000_000_000_000_000),
+    integers(min_value=1, max_value=100_000_000_000_000_000_000),
+    integers(min_value=1, max_value=100_000_000_000_000_000_000),
+)
+def test_fee_add_remove_invariant(flat_fee, prop_fee, imbalance_fee, amount, balance1, balance2):
+    """ First adding and then removing fees must yield the original value """
+    # Find examples where there is a reasonable chance of succeeding
```

Comment:
What does this comment mean? Examples of what? Succeeding in what?

Reply:
That's left over from the `test_fee_round_trip`, which I used as a starting point for this test, I'll change it.